packages/core/content-manager/admin/src/services/relations.ts

```diff
@@ -178,11 +177,8 @@ const relationsApi = contentManagerApi.injectEndpoints({
  * @description Adds a `__temp_key__` to each relation item. This gives us
  * a stable identifier regardless of it's ids etc. that we can then use for drag and drop.
  */
-const prepareTempKeys = (relations: RelResult[], existingRelations: RelationResult[] = []) => {
-  const [firstItem] = existingRelations.slice(0);
-
-  const keys = generateNKeysBetween(null, firstItem?.__temp_key__ ?? null, relations.length);
-
+const prepareTempKeys = (relations: RelResult[]) => {
+  const keys = generateNKeysBetween(null, null, relations.length);
```

Comment:
I have no doubt this works, but does it mean that when loading that appending new pages, the keys for the previously loaded pages get regenerated?

Asking because if the keys change it triggers re-renders that could be avoided (though it's not a huge deal)

Reply:
good point :) ! I will only regenerate the keys for new ones